ge/org.osate.ge.ba/src/org/osate/ge/ba/diagram/diagramType/BehaviorSpecificatonDiagramType.java

```diff
@@ -58,6 +60,8 @@ public boolean isApplicableToContext(final Object contextBo) {
 	public ImmutableSet<String> getDefaultContentFilters(final Object bo) {
 		if (bo instanceof BehaviorAnnex) {
 			return ImmutableSet.of(BehaviorVariableFilter.ID, BehaviorTransitionFilter.ID, BehaviorStateFilter.ID);
+		} else if (bo instanceof BehaviorTransition) {
```

Comment:
Let's avoid adding it to the default content filters for now. Connection labels layout are disabled because of an ELK exception and this will likely result in problems.